firestore/angular-rewrite/README.md

```diff
@@ -1,27 +1,59 @@
-`firebase --project demo-friendly-eats emulators:exec --import./exported-firestore [--export-on-exit=./exported-firestore]`
-(Only the `--export-on-exit` flag only if you want to persist changes after each run of the emulators. Otherwise, you will get a fresh dataset each execution.)
+FriendlyEats is a restaurant recommendation app built on Firestore. It was built with the [Angular](https://angular.io/) Javascript framework, and can make use of [Firebase's Local Emulator Suite](emulator-docs) (specifically, the [Authentication](https://firebase.google.com/docs/emulator-suite/connect_auth), [Storage](https://firebase.google.com/docs/emulator-suite/connect_storage), [Functions](https://firebase.google.com/docs/emulator-suite/connect_functions), and [Firestore](https://firebase.google.com/docs/emulator-suite/connect_firestore) emulators) so that the quickstart can be run without needing to create a project in the [Firebase Console](https://console.firebase.google.com).
 
-## Running the Angular App Only
+For more information about Firestore visit the [Firestore docs][firestore-docs].
+For more information about the Firebase Emulator Suite visit the [Emulator Suite docs][emulator-docs]
 
-Run `ng serve` to serve the Angular frontend without the emulated Firebase backend. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+[firestore-docs]: https://firebase.google.com/docs/firestore/
+[emulator-docs]: https://firebase.google.com/docs/emulator-suite
 
-## Code scaffolding
+## Prerequisites
+Before following the steps to build and run this quickstart you will need:
+ 1. [Node.Js](https://nodejs.org/en/download) Version 16.0 or higher
+ 2. [Java](https://jdk.java.net/) JDK version 11 or higher.
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+## Setup and run the app locally:
 
-## Build
+Follow these steps to setup and run the quickstart:
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
+ 1. Install the Angular CLI if you do not already have it installed on your machine:
+    ``` bash
+    npm install -g @angular/cli
+    ```
 
-## Further help
+ 1. Install the Firebase CLI if you do not have it installed on your machine:
+    ```bash
+    npm -g i firebase-tools
+    ```
+ 1. Initialize initialize / download the required emulators with the Firebase CLI
+    ```bash
+    firebase init emulators
+    ```
+    
+    When prompted, select the `Firestore`, `Storage`, `Functions`, and `Authentication` Emulators.
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+ 1. Install and build all project dependencies
+    ```bash
+    npm install && (cd functions && npm install && npm run build)
```

Comment:
is npm run build necessary? In the past I've skipped this step. Please test it out real quick and adjust if necessary

Reply:
It seems like running `npm run build` is necessary. I tested by cloning  #724 and seeing if I could run the functions and functions emulator without compiling them. While, running an initial  `npm run build` is neccesary, I've packaged this build into the `npm start` command (described in a comment below), so that this doesn't have to be done manually. Thus, I can get rid of `npm run build` here.